src/testdrive/src/action/kafka/ingest.rs

```diff
@@ -39,6 +40,7 @@ pub struct IngestAction {
     rows: Vec<String>,
     start_iteration: isize,
     repeat: isize,
+    headers: Vec<(String, Option<String>)>,
```

Comment:
Probably want this to be `Option<Vec<_>>` so we can test the case of empty headers explicitly.